artifactory/commands/transferfiles/transferfileprogress.go

```diff
@@ -235,6 +265,10 @@ func (t *TransferProgressMng) SetRunningThreads(n int) {
 	}
 }
 
+func (t *TransferProgressMng) IncreaseTotalSize(n int) {
```

Comment:
Looks like this function can be made provate.

Reply:
Done!